Rapportering/Remittentstod/constants.py

```diff
@@ -0,0 +1,299 @@
+from pathlib import Path
+import pandas as pd
+
+from rembox_integration_tools.rembox_analysis import StudyColumn, SeriesColumn
+
+VALID_STUDY_COLUMNS = StudyColumn()
+VALID_SERIES_COLUMNS = SeriesColumn()
+
+CLIENT_ID_ENV_VAR = "REMBOX_INT_CLIENT_ID"
+CLIENT_PWD_ENV_VAR = "REMBOX_INT_CLIENT_PWD"
+
+TOKEN_URI = "https://autoqa.vll.se/dpqaauth/connect/token"
+API_URI = "https://rembox.vll.se/api"
+ORIGIN_URI = "https://rembox.vll.se"
+
+MODALITY_CT: str = "CT"
+MODALITY_DX: str = "DX"
+MODALITY_MG: str = "MG"
+MODALITY_XA: str = "XA"
+
+MG_COL_EXAM_INDEX = "ExamIndex"
+MG_COL_PROJECTION = "Projection"
+MG_COL_EXAM_TYPE = "ExamType"
+
+MG_PROJ_LMLO = "LMLO"
+MG_PROJ_RMLO = "RMLO"
+MG_PROJ_LML = "LML"
+MG_PROJ_RML = "RML"
+MG_PROJ_LCC = "LCC"
+MG_PROJ_RCC = "RCC"
+
+MG_COMPRESSION_THICKNESS_RANGE = (40, 60)
+MG_SERIES_COUNT_FILTER = 4
+
+COL_MARKER_LINE_WIDTH = "markerLineWidth"
+
+REPORT_OUTPUT_DIR: Path = Path(__file__).parent / "Reports"
+
+MODALITY_LIST = [
+#    MODALITY_CT,
```

Comment:
Borde vi plocka bort kommentarerna på de andra modaliteterna innan vi trycker in i master?